docs/configuration/install.md

```diff
@@ -361,108 +333,169 @@ mysql -i -BN -e 'SELECT @@sql_mode' | grep -E 'ONLY_FULL_GROUP_BY|STRICT_TRANS_T
+You can [download](http://www.ilias.de/docu/goto.php?target=st_229) the latest
+ILIAS release or clone it from [GitHub](https://github.com/ILIAS-eLearning/ILIAS).
+For production use make sure to checkout the latest stable release, not the trunk,
+which is the development branch of the repository.
 
-### WebDAV with Windows Explorer
-Because of a special behaviour in the Windows Explorer, it sometimes fails to add a WebDAV connection with the error code "0x80070043 The Network Name Cannot Be Found".
+We recommend to clone from GitHub and use git to update the code, since this simplifies
+the update to future releases and versions.
 
-To prevent this behaviour, add the following rewrite rules to a .htaccess file in your webroot or to the corresponding section of the configuration of your webserver:
+Clone the code to the web servers docroot (e.g. `/var/www/html`) with the following
+commands:
 
 ```
-RewriteCond %{HTTP_USER_AGENT} ^(DavClnt)$
-RewriteCond %{REQUEST_METHOD} ^(OPTIONS)$
-RewriteRule .* "-" [R=401,L]
+cd /var/www/html/
+git clone https://github.com/ILIAS-eLearning/ILIAS.git .
+git checkout release_7
 ```
 
-### WebDAV with Mac Finder
-To upload files, the WebDAV Client *Finder* on Mac uses chunked transfer encoding. Some webservers can't handle this way of uploading files and are serving ILIAS an empty files, which results in an empty file object on ILIAS. Due to a bug in apache, the configuration of **Apache with PHP-FPM** does not work with the *Mac Finder*. If you use WebDAV on your ILIAS installation, we recommend to either use **Apache with mod_php** or **Nginx with PHP-FPM (> 1.3.8)**.
-
-<a name="install-other-depedencies"></a>
-## Install other Depedencies
+or unpack the downloaded archieve to the docroot
 
-On Debian/Ubuntu 14.04 execute:
-```
-apt-get install zip unzip imagemagick openjdk-7-jdk
-```
+The repository of ILIAS doesn't contain all code that is required to run. To
+download the required PHP-dependencies and to create static artifacts from
+the source, run the following in your ILIAS folder:
 
-On Ubuntu 16.04 execute:
 ```
-apt-get install zip unzip imagemagick openjdk-8-jdk
+composer install --no-dev
 ```
 
-On RHEL/CentOS execute:
-```
-yum install zip unzip libxslt ImageMagick java-1.8.0-openjdk
-```
-### Optional Dependencies
+This requires that the php dependency manager [composer](https://getcomposer.org/)
+is available in your $PATH.
 
-Depending on your use case, you MAY want to install further dependencies (exact package names vary by distribution):
+Create a directory outside the web servers docroot (e.g. `/var/www/files`). Make
+sure that the web server is the owner of the files and directories that were created
+by changing the group and owner to www-data (on Debian/Ubuntu) or apache (on RHEL).
+
+```
+chown www-data:www-data `/var/www/html
+chown www-data:www-data `/var/www/files
+```
+
+If you want to server other applications from your webserver, you might want to
```

Comment:
```suggestion
If you want to serve other applications from your webserver, you might want to
```
...but I would actually suggest to kick that part out and only keep "The commands above will directly serve ILIAS from the docroot." I don't see many good reasons to want to serve ILIAS from subdirectories. Certificates are free (or at least getting cheaper if you want e.Vs.) and subdomains seem by far more elegant to me.